### ISSUE TITLE
Fix accessibility issues

### ISSUE THREAD

**Issue by eliserichards** [mozilla-lockwise/lockwise-android#493]
Related to #225 

The following issues were found when running the AccessibilityScanner and TalkBack:

### Item List
Hostname titles are cut off with decending letters (e.g. "y", "j")
Question for @changecourse: should the size of the list items change if the text is bigger? Should there be more padding for the text?
<img width="250" alt="itemlist_allentries_title_small" src="https://user-images.githubusercontent.com/43795363/54051772-6f9df200-4197-11e9-91b6-343d30ea1c75.png">  <img width="250" alt="itemlist_allentries_title_large" src="https://user-images.githubusercontent.com/43795363/54051779-73ca0f80-4197-11e9-9674-1d49b95d2b2a.png"> 

Sort menu item titles are not visible with large text size
<img width="350" alt="itemlist_sortby_menu_large" src="https://user-images.githubusercontent.com/43795363/54051782-762c6980-4197-11e9-8f85-0cc254596698.png"> 

### Onboarding - Autofill
Button text is cut off with large text size
<img width="250" alt="onboarding_gotosettings_button_medium" src="https://user-images.githubusercontent.com/43795363/54051851-a70c9e80-4197-11e9-80bc-0f96274d935a.png">  <img width="250" alt="onboarding_gotosettings_button_large" src="https://user-images.githubusercontent.com/43795363/54051856-a96ef880-4197-11e9-8462-9cd4c895ad44.png">

**Comment by eliserichards** [comment]
Closing in favor of #800.
